src/transformers.ts

```diff
@@ -4,6 +4,8 @@ import get from 'dlv'
 import ldexp from '@stdlib/math-base-special-ldexp'
 import { dset } from 'dset';
 import { unset } from './unset'
+import crypto from 'crypto';
```

Comment:
Analytic.js depends on this, so this needs to be capable of being run in a browser.